src/EnergyPlus/PsychCacheData.hh

```diff
@@ -121,40 +163,40 @@ struct PsychrometricCacheData : BaseGlobalStruct
 {
 
 #ifdef EP_cache_PsyTwbFnTdbWPb
-    Array1D<cached_twb_t> cached_Twb; // DIMENSION(0:twbcache_size)
+    std::array<cached_twb_t, twbcache_size> cached_Twb;
 #endif
 #ifdef EP_cache_PsyPsatFnTemp
-    Array1D<cached_psat_t> cached_Psat; // DIMENSION(0:psatcache_size)
+    std::array<cached_psat_t, psatcache_size> cached_Psat;
 #endif
 #ifdef EP_cache_PsyTsatFnPb
-    Array1D<cached_tsat_pb> cached_Tsat; // DIMENSION(0:tsatcache_size)
+    std::array<cached_tsat_h_pb, tsatcache_size> cached_Tsat;
 #endif
 #ifdef EP_cache_PsyTsatFnHPb
-    Array1D<cached_tsat_h_pb> cached_Tsat_HPb; // DIMENSION(0:tsat_hbp_cache_size)
+    std::array<cached_tsat_h_pb, tsat_hbp_cache_size> cached_Tsat_HPb;
 #endif
 
 #ifdef EP_psych_stats
-    Array1D<Int64> NumTimesCalled = Array1D<Int64>(NumPsychMonitors, 0);
-    Array1D_int NumIterations = Array1D_int(NumPsychMonitors, 0);
+    std::array<std::int64_t, static_cast<int>(PsychrometricFunction::Num)> NumTimesCalled;
+    std::array<int, static_cast<int>(PsychrometricFunction::Num)> NumIterations;
 #endif
 
     void clear_state() override
     {
 #ifdef EP_cache_PsyTwbFnTdbWPb
-        cached_Twb.clear();
+        cached_Twb.fill(cached_twb_t());
 #endif
 #ifdef EP_cache_PsyPsatFnTemp
-        cached_Psat.clear();
+        cached_Psat.fill(cached_psat_t());
 #endif
 #ifdef EP_cache_PsyTsatFnPb
-        cached_Tsat.clear();
+        cached_Tsat.fill(cached_tsat_h_pb());
 #endif
 #ifdef EP_cache_PsyTsatFnHPb
-        cached_Tsat_HPb.clear();
+        cached_Tsat_HPb.fill(cached_tsat_h_pb());
```

Comment:
`std::array` does not have a `.clear()` method. So, I `.fill()`ed them with the default constructor call for those structs.